pkg/webhook/common.go

```diff
@@ -149,6 +149,10 @@ func (h *webhookHandler) tracingLevel(ctx context.Context, req *admission.Reques
 }
 
 func (h *webhookHandler) skipExcludedNamespace(req *admissionv1.AdmissionRequest, excludedProcess process.Process) (bool, error) {
+	if req.Operation == admissionv1.Delete && req.Object.Raw == nil {
```

Comment:
Thanks for the pr @clementnuss!

This PR https://github.com/open-policy-agent/gatekeeper/pull/3511/files moved `setObjectOnDelete` specifically it removed https://github.com/open-policy-agent/gatekeeper/pull/3511/files#diff-d3c8e52b699d3596d1057b82890a2ed8624445f1ee8f6d3d8a532e13766b99a8L146-L151 which would have updated `req` that would be used by skipExcludedNamespace. can you put it back and validate? Thanks!

Reply:
thanks for the pointer!
I added it back with my first commit, and in my second commit I reduced code duplication and used `util.SetObjectOnDelete` from the `target.go` file.

it works on my side, and all tests are passing. let me know if I should remove the 2nd commit or how you want it.